### PR TITLE
mob: init at v0.0.25

### PR DESCRIPTION
**Pull request by ericdallo**
###### Motivation for this change
Add mob (https://github.com/remotemobprogramming/mob)

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [X] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by thiagokokada — APPROVED** [review]

**Comment by jonringer** [comment]
@GrahamcOfBorg eval

**Comment by jonringer** [comment]
@GrahamcOfBorg eval

File: pkgs/applications/misc/mob/default.nix [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{ buildGoPackage, fetchFromGitHub, stdenv}:
```
Comment (dali99) [inline pkgs/applications/misc/mob/default.nix:1]:
```suggestion
{ lib, buildGoPackage, fetchFromGitHub }:
```

File: pkgs/applications/misc/mob/default.nix [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+buildGoPackage rec {
+  pname = "mob";
+  version = "0.0.25";
+  goPackagePath = "github.com/remotemobprogramming/mob";
+
+  src = fetchFromGitHub {
+    rev = "v${version}";
+    owner = "remotemobprogramming";
+    repo = pname;
+    sha256 = "sha256:1gs0mv4j66278srrck7llvi5gnfdqxz3a489qn5sx3mw46yx5q93";
```
Comment (dali99) [inline pkgs/applications/misc/mob/default.nix:12]:
```suggestion
    sha256 = "1gs0mv4j66278srrck7llvi5gnfdqxz3a489qn5sx3mw46yx5q93";
```

File: pkgs/applications/misc/mob/default.nix [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  };
+
+  meta = with stdenv.lib; {
```
Comment (dali99) [inline pkgs/applications/misc/mob/default.nix:15]:
```suggestion
  meta = with lib; {
```

**Review by D4ndellion — CHANGES_REQUESTED** [review]

**Comment by ericdallo** [comment]
Done, thanks @dali99

**Review by D4ndellion — APPROVED** [review]

**Comment by doronbehar** [comment]
Please fix merge conflicts.

**Comment by doronbehar** [comment]
@ofborg eval

**Comment by doronbehar** [comment]
@GrahamcOfBorg eval

**Comment by ericdallo** [comment]
Done @doronbehar

**Review by doronbehar — APPROVED** [review]

**Comment by doronbehar** [comment]
Result of `nixpkgs-review pr 96854` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>mob</li>
  </ul>
</details>

**Pull request merged**